homework/content.js

```diff
@@ -1,37 +1,44 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+	var NoteContentManager = function() {
+		this._wrapper = document.querySelector('#note-content-wrapper');
+	};
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+	NoteContentManager.prototype = {
+		start() {
+			window.addEventListener('note-open', function(event) {
```

Comment:
I prefer to use `addEventListener('note-open', this)` instead of binding on a function, since in this way all events could be redirected to `this.handleEvent`, and the `this` keyword in that method will refer to the instance you want to bind.

See this:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Example